gooddata-sdk/tests/catalog/test_catalog_organization.py

```diff
@@ -5,10 +5,13 @@
 
 from tests_support.vcrpy_utils import get_vcr
 
+from gooddata_api_client.exceptions import NotFoundException
 from gooddata_sdk import CatalogOrganization, GoodDataSdk
+from gooddata_sdk.catalog.organization.entity_model.jwk import CatalogJwk, CatalogJwkAttributes, CatalogRsaSpecification
```

Comment:
There should be 

`from gooddata_sdk import CatalogJwk, CatalogJwkAttributes, CatalogRsaSpecification` to be able to do that place `from gooddata_sdk.catalog.organization.entity_model.jwk import CatalogJwk, CatalogJwkAttributes, CatalogRsaSpecification` into [__init__.py](https://github.com/gooddata/gooddata-python-sdk/blob/master/gooddata-sdk/gooddata_sdk/__init__.py).